Modules/Core/Common/src/itkCommonEnums.cxx

```diff
@@ -175,6 +175,8 @@ operator<<(std::ostream & out, const CommonEnums::CellGeometry value)
         return "itk::CommonEnums::CellGeometry::VERTEX_CELL";
       case CommonEnums::CellGeometry::LINE_CELL:
         return "itk::CommonEnums::CellGeometry::LINE_CELL";
+      case CommonEnums::CellGeometry::POLY_LINE_CELL:
+        return "itk::CommonEnums::CellGeometry::POLY_LINE_CELL";
```

Comment:
Should we use `POLYLINE_CELL` instead of `POLY_LINE_CELL`? Is is similar to `POLYGON_CELL` defined below.

Reply:
When the appropriate name is decided, add it to the enumeration streaming test in here, please:
https://github.com/InsightSoftwareConsortium/ITK/blob/master/Modules/Core/Common/test/itkLoggerTest.cxx#L207

Thanks for doing this @PranjalSahu.